packages/safe-web3-lib/src/Web3Adapter.ts

```diff
@@ -187,6 +191,10 @@ class Web3Adapter implements EthAdapter {
   call(transaction: EthAdapterTransaction): Promise<string> {
     return this.#web3.eth.call(transaction)
   }
+
+  encodeParameter(type: string, value: any): string {
```

Comment:
What do you think about using `encodeParameters` instead of `encodeParameter` and accepting an array of types and values? If we proceed with this function if any time in the future, we need to implement the same function, but for encoding multiple parameters, we would either have to do a breaking change or add a new function

Reply:
https://web3js.readthedocs.io/en/v1.2.11/web3-eth-abi.html#encodeparameters